samples/SimpleDataSource/README.md

```diff
@@ -0,0 +1,30 @@
+# Introduction 
+The Performance Toolkit SDK allows for you to create AddIns to process your own data to be used by viewers like Windows Performance Analyzer (WPA). This project is a sample AddIn that you can use as a reference for creating your own AddIn. 
```

Comment:
Can we use "Plugin" (one capital) here, since that's the terminology we (I think?) want to adopt long term?